SCAutolib/controller.py

```diff
@@ -103,6 +104,16 @@ def setup_system(self, install_missing: bool, gdm: bool):
         if gdm:
             packages.append("gdm")
 
+        if graphical:
+            # install RPM fusion free for ffmpeg
+            version = int(run("rpm -E %rhel").stdout)
+            link = "https://mirrors.rpmfusion.org/" + \
```

Comment:
1) I would suggest to use condition or try/except block to execute this only if ffmpeg is not already installed 
2) Did you consider using EPEL instead of rpmfusion?

Reply:
As far as i know, ffmpeg is not available in EPEL repo. However, there is ffmpeg free, which should be the same program, just with less codecs. I will investigate this alternative.